src/plone/restapi/types/configure.zcml

```diff
@@ -19,7 +19,6 @@
         zcml:condition="installed z3c.relationfield"
         factory=".z3crelationadapter.ChoiceslessRelationListSchemaProvider"
         name="IRelatedItems.relatedItems" />
-    <adapter factory=".adapters.SubjectsFieldJsonSchemaProvider" name="IDublinCore.subjects" />
```

Comment:
@davisagli could you elaborate why the SubjectFieldJsonSchemaProvider has been removed here?